### PR TITLE
merge accelerators: update travis & README

### PR DESCRIPTION
**Pull request by abdulajet**
## Pull request checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I have resolved any merge conflicts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,11 +148,146 @@ OTOneToOneCommunicationController *vc = [OTOneToOneCommunicationController oneTo
...
+[self.annotator connectForSendingAnnotationWithSize:self.sharer.subscriberView.frame.size
+                                completionHandler:^(OTAnnotationSignal signal, NSError *error) {
+    
+                                    if (signal == OTAnnotationSessionDidConnect){
+        
+                                        // configure annotation view
+                                        self.annotator.annotationScrollView.frame = self.view.bounds;
+                                        [self.view addSubview:self.annotator.annotationScrollView];
+
+                                        // self.sharer.subscriberView is the screen shared from a remote client.
```
Comment (marinaserranomontes) [inline README.md:257]: Do we want to keep these comments?

File: README.md [hunk 2]
```diff
@@ -137,11 +148,146 @@ OTOneToOneCommunicationController *vc = [OTOneToOneCommunicationController oneTo
...
+                                        self.annotator.annotationScrollView.annotationView.currentAnnotatable = [OTAnnotationPath pathWithStrokeColor:[UIColor yellowColor]];
+                                    }
+                                }];
+
+self.annotator.dataSendingHandler = ^(NSArray *data, NSError *error) {
+    NSLog(@"%@", data);
+};
+```
+
+### Class design
```
Comment (marinaserranomontes) [inline README.md:272]: Should we add the Class design for TextChat and Core as well?

File: README.md [hunk 3]
```diff
@@ -150,7 +296,7 @@ To use OpenTok's framework you need a Session ID, a Token, and an API Key. You c
 
 ## Accelerator Core as a dependency
 
-The Accelerator Core is required whenever you use any of the OpenTok accelerators. The Accelerator Core is a common layer that includes the audio-video communication logic contained in all [OpenTok One-to-One Communication Sample Apps](https://github.com/opentok/one-to-one-sample-apps), and permits all accelerators and samples to share the same OpenTok session. The accelerator packs and sample app access the OpenTok session through the Common Accelerator Session Pack layer, which allows them to share a single OpenTok session:
+The Accelerator Core includes the audio-video communication logic contained in all [OpenTok One-to-One Communication Sample Apps](https://github.com/opentok/one-to-one-sample-apps). The accelerator and sample app access the OpenTok session through the Accelerator Session Pack layer, which allows them to share a single OpenTok session:
```
Comment (marinaserranomontes) [inline README.md:299]: I think https://github.com/opentok/one-to-one-sample-apps is deprecated. So maybe, we could skip it

File: README.md (hunk 2 above)
Comment (abdulajet) [reply README.md:272]: I would say that is out of scope for the merging, but we can create a ticket in general to improve the README. I just copied the other READMEs into here

**Review by marinaserranomontes — APPROVED** [review]
LGTM

**Pull request merged**